### DISCUSSION TITLE
Using Symbolic Math

### DISCUSSION THREAD

**Discussion by Ephraim-Bryski** [Ephraim-Bryski/seeciv#3] [Ideas]
I think it would be neat to allow the user to define systems of equations which can be solved and could relate to the graphics. I would want to use:

- MathQuill for the equation editor
- nerdamer for the CAS (or sympy)
- commands for substituting and solving which can be typed into the equation editor and will be interpreted

Attached is a simple example for projectile motion. Do you think the syntax is clear and intuitive? Any ideas on this and potential issues?
![20210409_131428](https://user-images.githubusercontent.com/38594736/114217531-4cc20700-9936-11eb-9682-517f5c50c8a6.jpg)

**Comment by officialabdulrehman** [comment]
It's clear enough
